### DISCUSSION TITLE
[文章页] 我的加载页面没有楼主快，缓存问题讨论

### DISCUSSION THREAD

**Discussion by Angel0726** [tangly1024/NotionNext#2070] [General]
https://jeweled-trilby-556.notion.site/bf19348f74cf470199d13aff0f4c5eb1
**描述bug**
简单说明bug的现象、相关的错误提示、日志等

**复现步骤**
出现这个bug的操作步骤

**期望的正常结果**
希望按这个步骤，正常操作结果是什么

**截图**
相关的页面，应该的结果

**环境**

- 操作系统: [例如. iOS, Android, macOS, windows]
- 浏览器 [例如. chrome, safari, firefox]
- NotionNext版本 [e.g. 3.13.6]

**补充说明**
与问题相关的其它说明

**Comment by sunheyi6** [comment]
@Angel0726 你的这个没有使用cdn加速的

**Comment by Angel0726** [comment]
> @Angel0726 你的这个没有使用cdn加速的

有免费好用的加速方案吗

**Comment by sunheyi6** [comment]
> > @Angel0726 你的这个没有使用cdn加速的
> 
> 有免费好用的加速方案吗

有的啊  我的是使用的自定义域名 然后cdn使用的是cloudflare

**Comment by Angel0726** [comment]
cloudfare经常卡顿的

**Comment by tangly1024** [comment]
路过... 这是一个复杂的问题，关于缓存预热。

可以明确的是：文章第二次被访问时，加载的速度就会非常快；

我也遇到：我的部分文章首次访问也会卡个几秒钟，但只要有一个访客浏览了此文章，这篇文章在被下一位访客浏览时就很快，因为这时候被缓存加速了。

注意：假设首页被访问了一次，此时首页上所有能被链接到的文章列表，都会一起被加速预热。即使你只访问了文章1，但列表中的文章2也因为你的此次访问被加速缓存了。

**Comment by Angel0726** [comment]
> 路过... 这是一个复杂的问题，关于缓存预热。
> 
> 可以明确的是：文章第二次被访问时，加载的速度就会非常快；
> 
> 我也遇到：我的部分文章首次访问也会卡个几秒钟，但只要有一个访客浏览了此文章，这篇文章在被下一位访客浏览时就很快，因为这时候被缓存加速了。
> 
> 注意：假设首页被访问了一次，此时首页上所有能被链接到的文章列表，都会一起被加速预热。即使你只访问了文章1，但列表中的文章2也因为你的此次访问被加速缓存了。

能让他缓存所有文章吗？
